deploy/1.8+/metrics-server-deployment.yaml

```diff
@@ -33,6 +33,8 @@ spec:
         args:
           - --cert-dir=/tmp
           - --secure-port=4443
+          - --kubelet-insecure-tls
```

Comment:
As this flag directly lowers security I would be against adding it as default.

Reply:
Thanks for your comments, I have applied the suggested args to lower the security but didn't work until I added the hostNetwork: true parameter.

Kubernetes version 1.16.1

Reply:
Sure, depending on your kubernetes distribution (how your kubelet and networking in configured), you would need to pick those flags. My major concern is that here we are proposing the default configuration which should not sacrifice security. 

I think we should add a better documentation about setting those flags in README in Deployment section. What do you think?

Reply:
My K8S was bootstrapped using kubeadm with default settings.

You might add a comment for education purposes only then to leverage monitoring metrics on kubernetes.

Please, fee free to close the PR.